### PR TITLE
fix(docs): Fix grammar overall

### PR DESCRIPTION
**Pull request by Moumi**
Fix grammar in the documentation files.

### CODE REVIEW

**Comment by Moumi** [comment]
Squashed the commits together.

File: docs/config/01-configuration-file.md [hunk 1]
```diff
@@ -239,8 +237,24 @@ Especially on services like SauceLabs and Browserstack it makes sense to only la
 **Default:** `null`
 
 **Description:** If `null` (default), uses karma's own `debug.html` file.
+>>>>>>> be1276d9ead4bc045a1adab9a80844be0422b2a3
```
Comment (dignifiedquire) [inline docs/config/01-configuration-file.md]: Merge issue?

File: docs/config/03-browsers.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ the browser will figure as `Chrome w/o security` in logs and reports.
 
 ## Correct path to browser binary
 Each plugin has some default paths where to find the browser binary on particular OS.
-You can override these settings by `<BROWSER>_BIN` ENV variable, or alternatively by creating a `symlink`.
+You can override these settings by `<BROWSER>_BIN` ENV variable, or by creating a `symlink`.
```
Comment (dignifiedquire) [inline docs/config/03-browsers.md]: this was fine before

File: docs/config/04-preprocessors.md [hunk 3]
```diff
@@ -119,7 +119,11 @@ preprocessors: {
 ```
 
 then for `a.js`, karma will run `'a'` then `'b'` then `'c'`.  If two lists
+<<<<<<< HEAD
```
Comment (dignifiedquire) [inline docs/config/04-preprocessors.md]: Merge issues

File: docs/config/04-preprocessors.md [hunk 4]
```diff
@@ -139,4 +143,8 @@ preprocessors: {
 ```
 
 Then `'a'` will definitely be run first, `'d'` will definitely be run last, but
+<<<<<<< HEAD
```
Comment (dignifiedquire) [inline docs/config/04-preprocessors.md]: Merge issues

**Comment by dignifiedquire** [comment]
Thanks, please change the commit message to `docs: ..` as `fix` is reserved for actual code fixes.

**Comment by Moumi** [comment]
@dignifiedquire: Sorry for the inconvenience. If not mistaken, all the comments you had provided are solved and the commit message has been changed. Hope this is how it should be.

**Comment by dignifiedquire** [comment]
Thanks :octocat:

**Pull request merged**